src/db/queries.js

```diff
@@ -110,6 +110,18 @@ const dbUpdateMonitorRecovered = async (id) => {
   return await handleDatabaseQuery(UPDATE_RECOVERY, errorMessage, id);
 };
 
+const dbDeleteMonitor = async (id) => {
+  const DELETE_MONITOR = `
+    DELETE FROM monitor
+    WHERE id = $1
+    RETURNING *
+  `;
+  const errorMessage = 'Unable to delete monitor in database.';
+
+  const rows = await handleDatabaseQuery(DELETE_MONITOR, errorMessage, id);
```

Comment:
So we send a query to the db with the error message already set and then, if the query returns an error, we can use the set message – is that what the logic here is?

Reply:
Yeah that's correct - checkout `handleDatabaseQuery` which is defined in this same file for further clarification.